static/templates/compose.hbs

```diff
@@ -56,6 +56,10 @@
         <div id="out-of-view-notification" class="notification-alert"></div>
         <div class="composition-area">
             <button type="button" class="close" id='compose_close' title="{{t 'Cancel compose' }} (Esc)">&times;</button>
+            <div id="compose_top_right">
+                <a role="button" class="expand_composebox_button fa fa-angle-up" aria-label="{{t 'Expand composebox' }}" tabindex=0 title="{{t 'Expand composebox' }}"></a>
+                <a role="button" class="collapse_composebox_button fa fa-angle-down" aria-label="{{t 'Collapse composebox' }}" tabindex=0 title="{{t 'Collapse composebox' }}"></a>
+            </div>
```

Comment:
These should be `button type="button"`, not `a` tags, since they're not links.  Fixing this will let you use the same CSS as you have for `#compose_close`.  

Also, is there a reason this div is after the `Cancel compose` button?  If you position them in the right order with respect to each other, you may be able to use `float: right` with some margins, rather than absolute positioning (which we generally want to avoid wherever possible)

Reply:
Yes, actually i copied the behavior of `#compose_close`, it used absolute positioning, so I used.
If I don't use absolute positioning and use only `float: right`, a weird space comes as shown.
![image](https://user-images.githubusercontent.com/56730716/124380081-7641c300-dcd8-11eb-8b84-7028114b7563.png)

Also, now I have moved the `#compose_close` also in this menu. This menu can have new buttons in future with ease now.

Reply:
OK; probably we can fix that but it seems fine to leave that idea for future work.